package.json

```diff
@@ -9,7 +9,7 @@
   },
   "scripts": {
     "pretest": "npm install",
-    "test": "mocha ./test || :"
+    "test": "karma start --single-run --browsers ChromeHeadless karma.conf.js"
```

Comment:
I think it's important to have the `||  :` for `os test` for some reason, but this is this repo ever going to be used by the `os` tool?

Reply:
nope, this is the one we had to pull out of the os tool or completely refactor the tests.